packages/e2e/test/web-extension/extension/ui.ts

```diff
@@ -68,6 +69,9 @@ document
   .querySelector<HTMLButtonElement>('#clearAllowList')!
   .addEventListener('click', backgroundServices.clearAllowList);
 
+// REVIEW: There might be a problem with this tracker. The background part, using createSupplyDistributionTracker,
```

Comment:
@mkazlauskas , I would like your opinion on this. Thank you!

Reply:
SupplyDistributionTracker is there entirely for demo purposes, so it's not very important. Maybe just leave a comment that it won't switch when you switch the wallet.